tests/lexer_test.py

```diff
@@ -5,15 +5,11 @@
 import unittest
 import unittest.mock
 
-
-# We need this because our tests and source files are not in the same directory
-sys.path.insert(0, 'aamg')
-
-import generator
+from aamg import generator
```

Comment:
God I was tired!
That's the low-level C programmer "I know how my system works" me.

You are so right!